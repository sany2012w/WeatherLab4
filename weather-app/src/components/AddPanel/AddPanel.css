.CitiesPanel {
  padding: 1% 2%;
  font-size: 25px;
  margin-top: 30px;
}

.CitiesPanel .CitiesText {
  font-family: 'Bitter', serif;
  color: #21224b;
  width: 50%;
  float: left;
  font-weight: bold;
}

.CitiesInput {
  padding: 1%;
  outline: none;
  font-family: 'Bitter', serif;
  color: #21224b;
  border: 2px #3d485b ridge;
  border-radius: 5px;
  margin-left: 10px;
  padding: 8px;
  font-size: 15px;
  font-weight: bold;
}

.CitiesButton {
  padding: 1%;
  outline: none;
  font-family: 'Bitter', serif;
  color: #21224b;
  border: 2px #3d485b ridge;
  border-radius: 5px;
  margin-left: 10px;
  padding: 8px;
  font-size: 15px;
  font-weight: bold;
}

.CitiesButton:hover {
  background-color: #586d7c;
  border-color: #c18ce3;
  color: white;
  cursor: pointer;
}

.CitiesButton:active {
  border-style: inset;
}
