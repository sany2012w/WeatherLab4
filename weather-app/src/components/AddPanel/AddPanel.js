import React from 'react';
import './AddPanel.css';

export default function AddPanel(props) {
  return (
    <div className="CitiesPanel">
      <div className="CitiesText">Favorites</div>
      <form  onSubmit={(e) => props.onSubmit(e)}>
        <input className="CitiesInput" type="text" name="cityName" placeholder="City name" required />
        <input className="CitiesButton" type="submit" value="Add city"/>
      </form>
    </div>
  );
}
