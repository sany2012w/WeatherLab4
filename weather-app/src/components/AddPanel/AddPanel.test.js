import React from "react";
import AddPanel from "./AddPanel";
import renderer from "react-test-renderer";

it(`AddPanel displayed correctly`, () => {
  const tree = renderer.create(<AddPanel />).toJSON();
  expect(tree).toMatchSnapshot();
});
