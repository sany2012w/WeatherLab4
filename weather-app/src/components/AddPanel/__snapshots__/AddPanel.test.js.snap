// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AddPanel displayed correctly 1`] = `
<div
  className="CitiesPanel"
>
  <div
    className="CitiesText"
  >
    Favorites
  </div>
  <form
    onSubmit={[Function]}
  >
    <input
      className="CitiesInput"
      name="cityName"
      placeholder="City name"
      required={true}
      type="text"
    />
    <input
      className="CitiesButton"
      type="submit"
      value="Add city"
    />
  </form>
</div>
`;
